qabelbox/src/main/res/values-de/strings.xml

```diff
@@ -191,4 +191,9 @@
     <string name="message_cant_attach_external_file_exists">Anfügen der freigegebenen Datei nicht möglich. Eine Datei mit diesem Namen existiert bereits. Vielleicht haben Sie diese Datei schon ein Mal früher angefügt</string>
     <string name="hockeyapp_feedback_response_button_text">Neue Nachricht senden</string>
     <string name="action_about">Über</string>
+   <string name="used_storage">Belegt: %1$s von %2$s</string>
+   <string name="unlimited_storage">unbegrentem Speicherplatz</string>
```

Comment:
Actually I am pretty sure that the case is grammatically correct. Even the extraordinary good spell check "duden.de" states the same "Belegt: 5 MB von unbegrenztem Speicherplatz." while "Belegt: 5 MB von unbegrenzt Speicherplatz" generates the error: "Überprüfen Sie, ob hier wegen der vorangehenden Präposition ein anderer Kasus (Fall) verlangt wird."

But maybe you just wanted to state that a "z" is missing. 

The more relevant question is, do we want to translate a quota of "0" as unlimited storage.